inc/namespace.php

```diff
@@ -17,6 +17,9 @@
  */
 function bootstrap() {
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_plugins', 1 );
+
+	// Remove AWS_Rekognition filter for performance purposes, as ElasticSearch is used instead.
+	remove_filter( 'posts_clauses', 'HM\\AWS_Rekognition\\filter_query_attachment_keywords' );
```

Comment:
The filter is added on `plugins_loaded` at priority 10 so this needs to be wrapped in an `add_action( 'plugins_loaded', ... , 11 );` call.

Reply:
Oh thank you! Didn't catch that one.